### PR TITLE
[IN-2025] initial prototype for iam policy management

### PR DESCRIPTION
**Pull request by sujithktkm**

### CODE REVIEW

File: terraform/environments/test/state.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+provider "aws" {
+  region  = "us-east-1"
+  version = "~> 2.43"
+}
+
+terraform {
+  required_version = "~> 0.12.16"
+  backend "s3" {
+    bucket         = "io-clari-tfstate"
+    key            = "policy-management/environments/test/state.tfstate"
```
Comment (sujithktkm) [inline terraform/environments/test/state.tf]: will need to update this to be in consistency with `arquiteto` references.

File: .gitignore [hunk 2]
```diff
@@ -11,8 +11,8 @@ crash.log
 # Ignore any .tfvars files that are generated automatically for each Terraform run. Most
 # .tfvars files are managed as part of configuration and so should be included in
```
Comment (RabiyaClari) [inline .gitignore]: so should *not* be included

File: terraform/environments/test/state.tf (hunk 1 above)
Comment (sujithktkm) [reply terraform/environments/test/state.tf]: fixed

File: .gitignore (hunk 2 above)
Comment (sujithktkm) [reply .gitignore]: this was auto generated and seems right to me. Please let me know otherwise

**Pull request merged**